data/helpers.d/backend

```diff
@@ -222,11 +222,7 @@ ynh_add_fpm_config () {
 
 	if [ -e "../conf/php-fpm.ini" ]
 	then
-		finalphpini="$fpm_config_dir/conf.d/20-$app.ini"
-		ynh_backup_if_checksum_is_different "$finalphpini"
-		sudo cp ../conf/php-fpm.ini "$finalphpini"
-		sudo chown root: "$finalphpini"
-		ynh_store_file_checksum "$finalphpini"
+		echo "Please do not use a separate ini file, merge you directives in the pool file instead." &>2
```

Comment:
Instead of just display an Error, i suggest something like this: 

ynh_print_debug "Please do not use a separate ini file, merge you directives in the pool file instead."
sed s/=/]=/g ../conf/php-fpm.ini | sed s/^/php_value[/g >> "$finalphpconf"

And to move ynh_store_file_checksum "$finalphpconf" at the end of the function.

Reply:
It's not as easy, it could be php_admin_value or php_admin_flag, I don't think playing with the pool file like that is really a good idea.

However, ignore the ini file is probably too drastic. We should maybe just add a warning to say that it should be modified before definitively removing it.

Reply:
I think too that to not try to change that dynamically is better. It will force all packager to adapt the app.

Reply:
ok